documentation/book/con-resource-RBAC.adoc

```diff
@@ -0,0 +1,12 @@
+[id='provision-rbac-{context}']
+= Provision Role-Based Access Control for the Cluster Operator
```

Comment:
Thsi seems to be also included in `documentation/book/con-configure-operator.adoc`.